docs/index.asciidoc

```diff
@@ -103,20 +102,20 @@ it, Set this parameter to 0
   * Value type is <<number,number>>
   * There is no default value for this setting.
 
-Force an random IV to be used per encryption invocation and specify
-the length of the random IV that will be generated via:
+In encryption operations, this plugin generates a random Initialization
+Vector (IV) per encryption operation, a standard best-practice to ensure
```

Comment:
```suggestion
Vector (IV) per encryption operation. This is a standard best-practice to ensure
```

Reply:
Note: It's not incorrect the way it is. Shorter sentences are easier for non-native speakers to parse and easier to translate.